File: Iniciante/1115 - Quadrante/JavaScript/1115.js
function main() {
  var input = require("fs").readFileSync("/dev/stdin", "utf8");
  var lines = input.split("\n");

  let controle = 0;
  for (let i = 0; controle === 0; i++) {
    let valorAtual = lines[i].split(" ");
    X = parseInt(valorAtual[0]);
    Y = parseInt(valorAtual[1]);

    if (X === 0 || Y === 0) {
      controle = 1;
    } else if (X > 0 && Y > 0) {
      console.log("primeiro");
    } else if (X > 0 && Y < 0) {
      console.log("quarto");
    } else if (X < 0 && Y < 0) {
      console.log("terceiro");
    } else if (X < 0 && Y > 0) {
      console.log("segundo");
    }
  }
}

main();

File: Iniciante/1115 - Quadrante/Python/1115.py
def main():
    parada = 0
    while parada == 0:
        X, Y = input().split()
        X = int(X)
        Y = int(Y)

        if X == 0 or Y == 0:
            parada = 1
        elif X > 0 and Y > 0:
            print("primeiro")
        elif X > 0 and Y < 0:
            print("quarto")
        elif X < 0 and Y < 0:
            print("terceiro")
        elif X < 0 and Y > 0:
            print("segundo")


main()
